cleanlab/rank.py

```diff
@@ -85,7 +87,7 @@ def order_label_issues(
 
     """
 
-    assert len(pred_probs) == len(labels)
+    assert_valid_inputs(X=None, y=labels, pred_probs=pred_probs, multi_label=False)
```

Comment:
if you hard code `multi_label=False` here (which i agree is correct), you also need to specify in the comments at the top and in the `labels` field that this function does not support multi_label (since rank does not support multi_label) and this method depends on rank to get the quality scores.

Reply:
That function never supported multi_label AFAICT, so it's not a change from this PR.

The docstring for `labels` already refers user to the docstring in `get_label_quality_scores()` which says multilabel classification is not supported.

Reply:
ah that works, although i think mentioning this at the end of the overall function header would be helpful to users instead of having to following a reference chain.

Reply:
as a user, i usually check the docstring of the function i'm calling while i'm writing code, and dont check the comment at the top of a module. Up to you. Agree it makes the docstrings redundant and verbose to include such things in every function header. I'm only recommending in this case since that line of code specifically assumes it and the assert will break if multi_labels are passed in.

Reply:
I'd prefer not to have half of a docstring for the function. So either it should copy the entire (long) docstring of the `labels` argument from `get_label_quality_scores()`, or just point user to that function's docstring as it currently does.  It's unclear with half a docstring why this would be the most relevant info to convey vs other info such as label formatting.